core/extraction/extraction_utils.py

```
@@ -43,18 +43,19 @@ def extract_decorators(node: ast.AST) -> List[str]:
     return decorators
 
 
-def extract_attributes(node: ast.ClassDef, source_code: str) -> List[Dict[str, Any]]:
-    """Extract class-level attributes.
+def extract_attributes(node: ast.ClassDef, source_code: str, instance: bool = False) -> List[Dict[str, Any]]:
```

Ruff (0.8.2)

Unused function argument: `source_code`

(ARG001)